aptos-move/block-executor/src/proptest_types/types.rs

```diff
@@ -466,6 +466,15 @@ where
             Transaction::Abort => ExecutionStatus::Abort(txn_idx as usize),
         }
     }
+
+    fn execute_transaction_sequential(
+        &self,
+        view: &impl TStateView<Key = K>,
+        txn: &Self::Txn,
+        txn_idx: TxnIndex,
+    ) -> ExecutionStatus<Self::Output, Self::Error> {
+        self.execute_transaction_parallel(view, txn, txn_idx, true)
```

Comment:
Is this intended?

Reply:
`execute_transaction_sequential` is supposed to be called only for sequential execution. 
`execute_transaction_parallel` is supposed to be called only for parallel execution. 
If this is confusion, I can change it to `execute_transaction_during_sequential_execution`. Should I change?

Reply:
maybe let's add that as a comment

Reply:
Done.